apps/pie-docs/src/_includes/content-page.njk

```diff
@@ -11,15 +11,19 @@
             <p class="c-contentPage-header-description">
                 {{ description }}
             </p>
+            {% if heroCtaTitle and heroCtaHref %}
+                <a class='c-contentPage-header-cta' href={{ heroCtaHref }}>{{ heroCtaTitle }}
```

Comment:
I've noticed this is an absolute link to prod on the preview, you might need to use the nunjucks `| url` filter here. Please look at other examples of it's usage in the project :)